src/common.scss

```diff
@@ -38,3 +38,7 @@ $whiteOpaque: rgba(255,255,255,0.5);
 
 $leafletControlBackgroundOpaque: rgba(255,255,255,0);
 $leafletControlBackgroundHover: rgba(255,255,255,0.5);
+
+$defaultWidth: 400px;
+$linkWidth: 500px;
+$nodeWidth: 450px;
```

Comment:
Since we have "NodeView" and "LinkView", I would call these "nodeViewWidth" or "linkViewWidth"